--- ./target/classes/META-INF/resources/webjars/openlayers/2.13.1/OpenLayers.debug.js	2013-07-09 19:31:14.000000000 +1000
+++ Heya/OpenLayers.debug.js	2016-02-18 15:28:34.580042780 +1100
@@ -64315,12 +64315,13 @@
                             cache.rendered = true;
                             me.setGMapVisibility(me.getVisibility());
                             me.moveTo(me.map.getCenter());
+                            cache.googleControl.appendChild(map.viewPortDiv);
                         });
                     } else {
-                        map.div.appendChild(container);
                         cache.googleControl.appendChild(map.viewPortDiv);
-                        google.maps.event.trigger(this.mapObject, 'resize');
                     }
+                    map.div.appendChild(container);
+                    google.maps.event.trigger(this.mapObject, 'resize');
                 }
                 this.mapObject.setMapTypeId(type);                
             } else if (cache.googleControl.hasChildNodes()) {
